gcp/README.md

```diff
@@ -673,3 +673,55 @@ resolved/update is done.
 ```
+
+#### External backups
+
+##### Setup.
+
+Create a Terragrunt definition like `gcp/live/prd/k8s/kube-system/backup-exporter/terraform.tfvars` inside the exekube project. This file contains 3 variables:
+   * `destination_bucket` - The destination GCS bucket, i.e "gs://gpii-backup-external-prd". 
+   * `replica_count` - the number of CouchDB replicas that the cluster has. This is important for copying all the snapshots of the cluster at the same time.
+   * `schedule` - Follows the same format as a Cron Job. i.e: `*/10 * * * *` to execute the task every 10 minutes.
+
+i.e:
+```
+# ↓ Module metadata
+
+terragrunt = {
+  terraform {
+    source = "/project/modules//backup-exporter"
+  }
+
+  dependencies {
+    paths = [
+      "../k8s-snapshots",
+    ]
+  }
+
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+destination_bucket = "gs://gpii-backup-dev-alfredo"
+replica_count      = 2
+schedule           = "*/30 * * * *"
+```
+
+##### Sending backup outside of the organization.
+
+The process is mostly executed by the `gcloud compute images export` command, which uses Cloud Build service, which uses [Daisy](ttps://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy) to create a VM, attach the images based on the snapshots to backup, create a file from those images and send the result files to the external storage.
+
+The destination bucket must be created manually. This is a once time setup for STG and PRD, because it doesn't need to be recreated in the future. The bucket must have the following permissions for the service account `[Source_project_number_id]@cloudbuild.gserviceaccount.com` needs the `Storage Admin` roles, because the Cloud Build service needs to copy the final images in the GCS bucket. Note that such service account must exist in order to attach the role. The Cloud Build service account is created when the Cloud Build API is enabled.
+
+##### Restoring a backup from outside of the organization.
+
+The process of the restore it's similar but the other way around. It uses the `gcloud compute images import` command. In order to make easier the process a rake task called `restore_snapshot_from_image_file`can be used. This task takes only one parameter with each snapshot to recover separated by one space. i.e:
+```
+rake restore_snapshot_from_image_file["gs://gpii-backup-dev-alfredo/2019-05-03_210008-pv-database-storage-couchdb-couchdb-0-030519-205726.raw.gz gs://gpii-backup-dev-alfredo/2019-05-03_210008-pv-database-storage-couchdb-couchdb-1-030519-205756.raw.gz"]
+```
+Once the task finished new restored snapshots will appear with the leading in the name `external-`. This will help with the search when at the restoration of the disks using the snapshots.
+
+And follow the [restore a pv from snapshot procedure](https://github.com/gpii-ops/gpii-infra/tree/master/gcp#data-corruption-on-a-single-couchdb-replica) using the snapshots restored.
```

Comment:
Is "corruption on a single couchdb replica" the right doc to link to?

Reply:
Fixed in https://github.com/gpii-ops/gpii-infra/pull/432/commits/d8b3176d7c88cc554c2da595e5203e9a602c7211